include/nuraft_mesg/mesg_state_mgr.hpp

```diff
@@ -28,6 +28,15 @@ struct replica_config {
     std::string aux;
 };
 
+struct peer_info {
+    // Peer ID.
+    std::string id_;
+    // The last log index that the peer has, from this server's point of view.
+    ulong last_log_idx_;
+    // The elapsed time since the last successful response from this peer, set to 0 on leader
+    ulong last_succ_resp_us_;
```

Comment:
This one is prolly fine I guess...what's the size restriction on whatever it's being marshaled into?

Reply:
same here ,will make it u64,  or probabaly u32 is enough(59days)  ?  I am not sure if a raft group is completely idle as no new writes(full) and no delete,  will anything internal still go through proposal?